src/seedu/addressbook/data/person/Address.java

```diff
@@ -28,6 +29,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         }
         this.value = trimmedAddress;
     }
+    
+    @Override
+    public String getPrintableString(Printable... printables) { 
```

Comment:
No need to keep mentioning `Printable... printables`, recollect from tutorial discussion.